deploy/ansible/roles/k8s_resources/tasks/kube-fluentd.yaml

```diff
@@ -51,6 +51,8 @@
     name: "{{ cluster_name }}-fluentd-role"
     trust_policy_filepath: "{{ tmp_dir }}/trust_policy.kube-fluentd.{{ cluster_name }}.yaml"
     state: present
+  retries: 10
```

Comment:
Can we only retry the task when stderr contains "Rate exceeded" string?
For example:
```yaml
registr: result
until: '"Rate exceeded" in result.msg'
```
And can we increase delay and timeout? IMHO, they are too small.

Reply:
There might be some other response from AWS missed in this case, like Connection timeout or smth like that.
Timeouts is fine I guess because the kimits are not so small and 5 sec should be enouugh to avoud race condition with other runs. Worth testing that anyway.